data/json/mapgen/animalpound.json

```diff
@@ -27,30 +27,29 @@
     "object": {
       "fill_ter": "t_floor",
       "rows": [
-        "''''''''''~~~~'''''*''''",
-        "''''*'''''~~~~'*''''''''",
-        "''---------++---------''",
-        "''-].y-]%C....D-lt-tl-''",
-        "''-%c.-]cC.....-ll-ll-''",
-        "''-d..-r.C....y-ls-sl-''",
-        "''---i-R.C.....-+---+-'*",
-        "''-<+................-''",
-        "*'---------++---------''",
-        "''-lllllll-ll-l|l|l|l-*'",
-        "''-======l-ll-^=^=^=^-''",
-        "''-l|l|l|l+ll+lllllll-''",
-        "''-a-a-a-l-ll---------''",
-        "''-Fllllil-ll-FFss#kk-''",
-        "''-a-a-a-l-ll-lllllll-''",
-        "''-l|l|l|l+ll-QlTllTl-f'",
-        "*'-======l-llill#ll#l-f'",
-        "''-lllllll-ll-------M-4'",
-        "''----------M-         '",
-        "''-R.......i   ,       '",
-        "''--a-a--a--   ,       '",
-        "''|  |  |  |   ,       '",
-        "'*|========|           '",
-        "''                    ''"
+        "     ,     ~~     ,     ",
+        "     ,     ~~     ,     ",
+        "     ,     ~~     ,     ",
+        "'~~~~~~~~~~~~~~~~~~~~~~'",
+        "'----------++----------'",
+        "'--].y-]%C......-lt-tl-'",
+        "'--%c.-]cC.....D-ll-ll-'",
+        "'--d..-r.C......-ls-sl-'",
+        "'----i-R.C.....y-+---+-'",
+        "'-<+..................-*",
+        "*----------++----------'",
+        "*-|=====|F-ll-FFss#kkk-'",
+        "'-|lllllal-ll-Qlllllll-*",
+        "'-|=====|l-ll+llTllTll-'",
+        "'-|lllllal-ll-ll#ll#ll-'",
+        "'-|=====|l+ll----------'",
+        "'-|lllllal-ll-FFss#kkk-'",
+        "'-|=====|l-ll-Qlllllll-'",
+        "'-|lllllal-ll+llTllTll-'",
+        "*-|=====|R-ll-ll#ll#ll-'",
+        "'----------M-----------4",
+        "'~~~~~~~~~~~~~~~~~~~~ff~",
+        "''''''''''''''''''''''''"
```

Comment:
There is a row missing. The original map had 24 rows, the new one only has 23 rows.